ietf/idindex/tasks.py

```diff
@@ -43,9 +49,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 @shared_task
 def idindex_update_task():
     """Update I-D indexes"""
+    # Why are these not using values from django.conf.settings?
```

Comment:
😵‍💫 

They should - probably leftover from when this was shell code in `bin/hourly` and copied over too-faithfully